### DISCUSSION TITLE
Error 404

### DISCUSSION THREAD

**Discussion by localcake** [victornpb/undiscord#280] [1. Questions & Answers] [answered]
![image](https://user-images.githubusercontent.com/88810043/145623726-1b1e5317-7ef9-4f22-bd13-ac04e191c521.png)
I just got it and this happens

**Comment by KitsuTheFloof** [comment]
I have this issue too

**Reply by luni3359** [reply]
Did your issue get solved?

**Comment by reddaugherty** [comment, marked as answer]
Looks like if you check discordstatus it shows that there is some maintenance going on with discord search which might be related?

https://discordstatus.com/

**Reply by KitsuTheFloof** [reply]
Thank you for letting me know

**Comment by firebird631** [comment]
Probably like you abuse out of the rate limit, and temporary ban. Try with a greater delay the next time.
